content/docs/concepts/traffic-management/overview/index.md

```diff
@@ -0,0 +1,155 @@
+
+{{< image width="40%"
+    link="./pilot-arch.svg"
+    caption="Pilot architecture"
+    >}}
+
+As the diagram illustrates, Pilot maintains an **abstract model** of all the
+services in the mesh. **Platform-specific adapters** in Pilot translate the
+abstract model appropriately for your platform.
+
+For example, the Kubernetes adapter implements controllers to watch the
+Kubernetes API server for changes to pod registration information, ingress
+resources, and third-party resources that store traffic management rules. The
+Kubernetes adapter translates this data for the abstract model, so Pilot can
+generate and deliver the appropriate Envoy-specific configurations.
+
+The Pilot **service discovery and traffic rules** use the abstract model to let
+Envoy proxies know about one another in the mesh through the **Envoy API.**
+
+You can use the **Networking and Rules APIs** to exercise more granular control
+over the traffic in your service mesh.
+
+## Envoy proxies
+
+Traffic in Istio is categorized as data plane traffic and control plane
+traffic. Data plane traffic refers to the data that the business logic of the
+workloads manipulate. Control plane traffic refers to configuration and control
+data sent between Istio components to program the behavior of the mesh. When
+referring to traffic management in Istio, the traffic involved is exclusively
+data plane traffic.
+
+Envoy proxies are the only Istio components that interact with data plane
+traffic. Envoy proxies route the data plane traffic across the mesh and enforce
+the configurations and traffic rules without the services having to be aware of
+them. Envoy proxies mediate all inbound and outbound traffic for all services
+in the mesh. Envoy proxies are deployed as sidecars to services, logically
+augmenting the services with traffic management features, including the two
+discussed in this overview:
+
+-  [Service discovery](/docs/concepts/traffic-management/overview/#discovery)
+-  [Load balancing](/docs/concepts/traffic-management/overview/#load-balancing)
+
+The [traffic routing and configuration](/docs/concepts/traffic-management/routing/)
+and [network resilience and testing](/docs/concepts/traffic-management/network/)
+sections dig into more sophisticated features and tasks enabled by Envoy
+proxies, which include:
+
+-  Traffic management features: enforce fine-grained traffic control with rich
+   routing rules for HTTP, gRPC, WebSocket, and TCP traffic.
+
+-  Network resiliency features: setup retries, failovers, circuit breakers, and
+   fault injection.
+
+-  Security and authentication features: enforce security policies and enforce
+   access control and rate limiting defined through the configuration API.
+
+### Platform-agnostic service discovery {#discovery}
+
+Service discovery works in a similar way regardless of what platform you're
+using:
+
+1. The platform starts a new instance of a service and notifies its platform
```

Comment:
It's pull, not push, isn't it?  The platform adapter is watching the platform

Reply:
@frankbu @andraxylia Do you concur that it's pull not push?

Reply:
It's just an issue of wording IMO. The instance start triggers a notification. So, I would suggest
```
1. The platform starts a new instance of a service which notifies the platform
```